packages/core/src/app/payment/paymentMethod/HostedWidgetPaymentMethod.tsx

```diff
@@ -332,9 +338,30 @@ class HostedWidgetPaymentMethod extends Component<
         );
     }
 
+
+    private isInstrumentFeatureAvailable() {
```

Comment:
Do we need to create this method here? Seems like we can call the method in `storedInstruments` folder directly?

Reply:
Yes, it can be called directly. I will change it.